lib/sles4sap_publiccloud.pm

```diff
@@ -429,7 +427,7 @@ sub stop_hana {
             $exit_code = script_run('nc -vz -w 1 ' . $self->{my_instance}->{public_ip} . ' 22', quiet => 1);
             $nc_error_occurrence += 1 if (!defined($exit_code) or $exit_code ne 0);
         }
-        my $end_msg = join("\n",
+        my $end_msg = join('\n',
```

Comment:
Do we want here a newline or the string `\n`? If it's the former, then we need double-quotes here.

Reply:
Quotes confusion. TY